control/control.rnc

```diff
@@ -470,6 +475,12 @@ partitioning_elements =
     | root_subvolume_read_only
     | subvolumes
 
+# you can either have the new ('ng' style elements or the old 'legacy' ones)
```

Comment:
NP: The position of brackets looks wrong.

`# you can either have the new 'ng' style elements or the old 'legacy' ones`